src/systempolicy/systemPolicy.go

```diff
@@ -459,25 +459,31 @@ func clusteringSystemLogsByNamespacePod(logs []types.KnoxSystemLog) map[SysLogKe
 	return results
 }
 
-func systemLogDeduplication(logs []types.KnoxSystemLog) []types.KnoxSystemLog {
+func systemLogDeduplication(logMap map[types.KnoxSystemLog]bool) []types.KnoxSystemLog {
```

Comment:
This function no longer does deduplication. This function just clean the logMap based on the two conditions above. So we can rename it to..
```suggestion
func cleanupSystemLog(logMap map[types.KnoxSystemLog]bool) []types.KnoxSystemLog {
```